es/chain/node.js

```diff
@@ -52,27 +53,21 @@ async function sendTransaction (tx, options = {}) {
     const { txHash } = await this.api.postTransaction({ tx })
 
     if (waitMined) {
-      const txData = { ...(await this.poll(txHash, options)), rawTx: tx }
+      const txData = { ...await this.poll(txHash, options), rawTx: tx }
       // wait for transaction confirmation
       if (options.confirm) {
         return { ...txData, confirmationHeight: await this.waitForTxConfirm(txHash, options) }
       }
       return txData
     }
     return { hash: txHash, rawTx: tx }
-  } catch (e) {
-    throw Object.assign(
-      (new Error(e.message)),
-      {
-        rawTx: tx,
-        verifyTx: () => this.unpackAndVerify(tx)
-      }
-    )
+  } catch (error) {
+    throw Object.assign(error, { rawTx: tx, verifyTx: () => this.unpackAndVerify(tx) })
   }
 }
 
 async function waitForTxConfirm (txHash, options = { confirm: 3 }) {
-  options.confirm = typeof options.confirm === 'boolean' && options.confirm ? 3 : options.confirm
+  options.confirm = options.confirm === true ? 3 : options.confirm
```

Comment:
I know this is an inherited logic, but is there any specific reason why the default confirmations to be awaited is `3`? 🤔

Reply:
I'm not sure. It like an arbitrary default, as waiting for 10 blocks in the poll function: https://github.com/aeternity/aepp-sdk-js/blob/df93448824e543cf0b00d285e8d37c84ddd81cd7/es/chain/node.js#L125